tetris.py

```diff
@@ -80,15 +85,27 @@ def draw_block(self):
 
         self.canvas.delete("block")
         for x, y in (
-            self.tetris_game.get_current_block() + self.tetris_game.landed_blocks
+            self.tetris_game.get_current_block()
         ):
             self.canvas.create_rectangle(
                 x * square_size,
                 y * square_size,
                 x * square_size + square_size,
                 y * square_size + square_size,
                 tags="block",
-                fill=RED,
+                fill=color_dict[self.tetris_game.current_block_shape]
+            )
+
+        for shape_letter, (x, y) in self.tetris_game.landed_blocks:
+            print(shape_letter, (x, y))
+            color = color_dict[shape_letter]
+            self.canvas.create_rectangle(
+                x * square_size,
+                y * square_size,
+                x * square_size + square_size,
+                y * square_size + square_size,
+                tags="block",
+                fill=color
```

Comment:
The review comment from #19 applies here too: if this copy/pasta bothers you, you can move it into a separate method.